src/psyclone/tests/test_files/gocean1p0/single_invoke_two_identical_kernels.f90

```diff
@@ -0,0 +1,55 @@
+!-------------------------------------------------------------------------------
```

Comment:
Please could you replace this copyright notice with the full one, see e.g. `test28_invoke_kernel_stencil.f90`